### PR TITLE
FRONT-1055: Minimalistic payloads

### PR DESCRIPTION
**Pull request by mondoreale**
Few things happen in this PR:
- Message payloads are only about the content of the message
  This gives us the ability, if allowed by the client, to display encrypted messages. Long story.
- We utilize the main-to-delegated address registry to read who's the real sender behind a message.
- Display sender ENS names/nicknames/addresses in the UI so that we can see who's the sender.
- Few other minor things I.. forgot to remember.

### CODE REVIEW

File: src/components/modals/EditMembersModal.tsx [hunk 1]
```diff
@@ -202,24 +202,22 @@ export default function EditMembersModal({ open, canModifyMembers = false, ...pr
...
-                                        isCurrentDelegatedAccount={isSameAddress(
-                                            address,
-                                            delegatedAccount
-                                        )}
-                                        permissions={permissions}
-                                        accountType={accountType}
-                                    />
-                                ) : null
-                            })}
+                            {members.map(({ address, permissions, accountType }) => (
```
Comment (mondoreale) [inline src/components/modals/EditMembersModal.tsx:205]: @bearni95, `members` no longer holds delegated accounts (you can see that logic in `detect.saga`). Now we don't have to filter out anything here.

File: src/features/members/sagas/detect.saga.ts [hunk 2]
```diff
@@ -36,6 +37,20 @@ function* onDetectAction({
...
+                    MembersAction.lookupDelegation({
+                        delegated: assignment.user,
+                        provider,
+                        fingerprint: Flag.isLookingUpDelegation(assignment.user),
+                    })
+                )
+
+                // Exclude delegated accounts from the members list. When we delete a main/unset
+                // accounts from a room we're gonna fetch the list of delegatees anyway (@TODO).
+                continue
```
Comment (mondoreale) [inline src/features/members/sagas/detect.saga.ts:51]: Here, the thing I mentioned about not having delegated accounts in `members`.

File: src/hooks/useIntercept.ts [hunk 3]
```diff
@@ -51,22 +52,12 @@ export default function useIntercept(roomId: RoomId) {
...
 
-            onMessage({
-                createdAt,
-                createdBy,
-                content,
-                updatedAt: createdAt,
-                id,
-                roomId,
-            })
+            onMessage(toLocalMessage(raw))
```
Comment (mondoreale) [inline src/hooks/useIntercept.ts:60]: Isn't it lovely? ;)

File: src/utils/db.ts [hunk 4]
```diff
@@ -61,7 +62,7 @@ async function teardown() {
 
 // If you want people to lose their local histories in previous versions uncomment
 // the following line:
-// teardown()
+teardown()
```
Comment (mondoreale) [inline src/utils/db.ts:65]: This will make old dbs go.

**Comment by mondoreale** [comment]
@bearni95 can you run it locally, tell me if it works for you before I click through getting it live?

**Review by bearni95 — APPROVED** [review]

**Pull request merged**